labs/vstsextend/sonarcloud/readme.md

```diff
@@ -211,7 +211,7 @@ The page has other metrics such as **_Code Smells_**, **_Coverage_**, **_Duplica
 | **Duplications**    | The duplications decoration shows which parts of the source code are duplicated                                                                                                                                                                                                           |
 | **Size**            | Provides the count of lines of code within the project including the number of statements, Functions, Classes, Files and Directories                                                                                                                                                      |
 
-{% include important.html content= "In this example, along with the bug count, a character **C** is displayed which is known as **Reliability Rating**. **C** indicates that there is **at least 1 major bug** in this code. For more information on Reliability Rating, click [here](https://docs.sonarqube.org/display/SONAR/Metric+Definitions#MetricDefinitions-Reliability). For more information on rule types and severities, see [here](https://docs.sonarqube.org/display/SONAR/Rules+-+types+and+severities)." %}
+{% include important.html content= "In this example, along with the bug count, a character **C** is displayed which is known as **Reliability Rating**. **C** indicates that there is **at least 1 major bug** in this code. For more information on Reliability Rating, click [here](https://docs.sonarqube.org/display/SONAR/Metric+Definitions#MetricDefinitions-Reliability). For more information on rule types see [here](https://docs.sonarqube.org/latest/user-guide/rules/) and for more inforamtion on severities, see [here](https://docs.sonarqube.org/latest/user-guide/issues/)." %}
```

Comment:
The `types and severities` link to our docs was broken. It looks like there are now two separate pages.